package/python-kiwi-spec-template

```diff
@@ -244,6 +244,7 @@ Requires:       lvm2
 Requires:       parted
 Requires:       kpartx
 Requires:       cryptsetup
+Requires:       mdadm
```

Comment:
Shouldn't this be for disk images and not filesystems?

Reply:
Yes, I agree that we should. These tools don't work if you're just creating plain filesystems anyway.